legal-api/src/legal_api/resources/v2/minio.py

```diff
@@ -0,0 +1,33 @@
+from flask import Blueprint, request, jsonify, current_app
+from flask_cors import cross_origin
+from http import HTTPStatus
+from legal_api.services.minio import MinioService
+from minio.error import S3Error
+from legal_api.utils.auth import jwt
+from legal_api.models import Filing
+
+bp = Blueprint('minio', __name__, url_prefix='/api/v2/documents')
+
+def is_complete_filing(filing_id: int) -> bool:
+    """Check if the filing is complete."""
+    filing = Filing.query.get(filing_id)
+    return filing and filing.status == 'COMPLETED'
+
```

Comment:
Not sure most optimal way to check if completed filing

Could query here and not use filing_id?

Reply:
Get filing_id from documents table (query with file_key) to verify its in DRAFT